src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs

```diff
@@ -140,7 +140,7 @@ public static object ConvertObjectToType(object objectToBeConverted, Type typeOf
                     }
                     else
                     {
-                        throw new ArgumentException(string.Format("'{0}' is not referencd but not defined in the file '{1}'", dep.Name, package.FilePath));
+                        throw new ArgumentException(string.Format("'{0}' is referencd but not defined in the file '{1}'", dep.Name, package.FilePath));
```

Comment:
There's also a typo that should be fixed while you're in there:  `referencd` should be `referenced`.